### ISSUE TITLE
Specify a validation set instead of a random validation portion

### ISSUE THREAD

**Issue by wpm** [wpm/mycroft#31]
These two arguments would be mutually exclusive.

Want this for the Stanford Sentiment Treebank because that experimental setup specifies train, dev, and test sets.
